src/cart/utils/crt_utils.c

```diff
@@ -375,8 +375,9 @@ crtu_dc_mgmt_net_cfg_rank_add(const char *name, crt_group_t *group,
 				true /* all_ranks */,
 				&crt_net_cfg_info,
 				&crt_net_cfg_resp);
-	if (opts.assert_on_error)
+	if (opts.assert_on_error) {
```

Comment:
even though i prefer this style and long time ago i had advocated for it, but this is not our coding style. If we want to change that, it's better to make the proposal at the pdt meeting before you do the change in everyone's code yourself.

Reply:
This is needed to prevent compile warnings, if D_ASSERT() is defined out then this code expands to the below
`if (cond)
    ;
`
which the compiler doesn't like as it's easy to make mistakes that way.  Adding in the braces makes it explicit that the conditional is expected and therefore silences the compiler warning.  It would have been possible to add -Wempty-body to silence these however on reflection I thought that the compiler warning was a reasonable one, hence why I changed the code where necessary.